File: 1-codes/micro/micro07.c
#import<stdio.h>

int main() {
  int numero = 0;
  int programa = 1;
  char opc;

  while (programa == 1) {
    printf("Digite um numero: ");
    scanf("%d", &numero);

    if(numero > 0) {
      printf("Positivo");
    } else {
      if (numero == 0) {
        printf("O numero e igual a 0");
      } else if (numero < 0) {
        printf("Negativo");
      }
    }
    printf("Deseja Finalizar? (S/N) ");
    scanf("%c", &opc);
    if (opc == 'S') {
      programa = 0;
    }
  }
  return 0;
}

File: 6-gerador/tests/micro07.py
def main() -> None:
    numero = 0
    programa = 1
    opc = ""
    while programa == 1:
        print("Digite um numero: ")
        int(input(numero))
        if numero > 0:
            print("\nPositivo")
        else:
            if numero == 0:
                print("\nO numero e igual a 0")
            elif numero < 0:
                print("\nNegativo")

        print("\nDeseja Finalizar? (S/N) ")
        str(input(opc))
        if opc == "S":
            programa = 0

main()
